docs/fundamentals/code-analysis/quality-rules/ca1012.md

```diff
@@ -1,6 +1,6 @@
 ---
-title: "CA1012: Abstract types should not have constructors (code analysis)"
-description: "Learn about code analysis rule CA1012: Abstract types should not have constructors"
+title: "CA1012: Abstract types should not have public constructors (code analysis)"
```

Comment:
Can you update lines 17 and 27 too?

Reply:
Ops, sorry for missing those. Will update now.